packages/mdc-drawer/_mixins.scss

```diff
@@ -147,3 +147,15 @@
   @include mdc-drawer-header_;
   @include mdc-drawer-list_;
 }
+
+@mixin mdc-drawer-fill-color($color) {
+  background-color: $color;
```

Comment:
We typically use the `mdc-theme-prop` mixin for these component theme mixins, because it supports our special variable names like `primary`, `secondary-light`, `text-hint-on-dark`, etc. in addition to color literals like `#aaccdd`.

Reply:
fixed